### PR TITLE
command_line: Fix typos and make table column size equal

### PR DESCRIPTION
**Pull request by OnkarRuikar**
## Summary
Fixed typos.

The table "list of pros and cons for globally installing" has different column size which looks odd:
> before

After fix:
> after

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for your PR, @OnkarRuikar !

I don't think this should be a table at all. The cells in each row don't have any relation to each other, for example "Only install once" and " Other developers in your team won't have access to these tools, for example if you are sharing the codebase over a tool like git.".

Semantically, it's two lists, one of pros and one of cons, and it should be formatted like that.

**Comment by OnkarRuikar** [comment]
> Semantically, it's two lists, one of pros and one of cons, and it should be formatted like that.

@wbamberg I've separated the lists.

File: files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 1]
```diff
@@ -337,44 +337,21 @@ Before we dive into installing Prettier, there's a question to answer — "where
 
 With `npm` we have the choice of installing tools globally — so we can access them anywhere — or locally to the current project directory.
 
-There's pros and cons each way — and this list of pros and cons for globally installing is far from exhaustive:
+There are pros and cons each way — and following lists of pros and cons for globally installing are far from exhaustive.
```
Comment (wbamberg) [inline files/en-us/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:340]:
```suggestion
There are pros and cons each way — and the following lists of pros and cons for globally installing are far from exhaustive.
```

**Review by wbamberg — APPROVED** [review]
This looks so much better, thank you @OnkarRuikar !

**Pull request merged**